packages/commons/src/risk-analysis/models.ts

```diff
@@ -75,17 +76,24 @@ export function riskAnalysisValidatedFormToNewRiskAnalysis(
     id: generateId<RiskAnalysisId>(),
     name,
     createdAt: new Date(),
-    riskAnalysisForm: {
-      id: generateId<RiskAnalysisFormId>(),
-      version: validatedForm.version,
-      singleAnswers: validatedForm.singleAnswers.map((a) => ({
-        ...a,
-        id: generateId<RiskAnalysisSingleAnswerId>(),
-      })),
-      multiAnswers: validatedForm.multiAnswers.map((a) => ({
-        ...a,
-        id: generateId<RiskAnalysisMultiAnswerId>(),
-      })),
-    },
+    riskAnalysisForm:
+      riskAnalysisValidatedFormToNewRiskAnalysisForm(validatedForm),
+  };
+}
+
+export function riskAnalysisValidatedFormToNewRiskAnalysisForm(
+  validatedForm: RiskAnalysisValidatedForm
+): RiskAnalysisForm {
+  return {
+    id: generateId<RiskAnalysisFormId>(),
+    version: validatedForm.version,
+    singleAnswers: validatedForm.singleAnswers.map((a) => ({
+      ...a,
+      id: generateId<RiskAnalysisSingleAnswerId>(),
+    })),
+    multiAnswers: validatedForm.multiAnswers.map((a) => ({
+      ...a,
+      id: generateId<RiskAnalysisMultiAnswerId>(),
+    })),
   };
 }
```

Comment:
arrow syntax?
```suggestion 
export riskAnalysisValidatedFormToNewRiskAnalysisForm = (
  validatedForm: RiskAnalysisValidatedForm
): RiskAnalysisForm => ({
    id: generateId<RiskAnalysisFormId>(),
    version: validatedForm.version,
    singleAnswers: validatedForm.singleAnswers.map((a) => ({
      ...a,
      id: generateId<RiskAnalysisSingleAnswerId>(),
    })),
    multiAnswers: validatedForm.multiAnswers.map((a) => ({
      ...a,
      id: generateId<RiskAnalysisMultiAnswerId>(),
    })),
  });
```

Reply:
I know the "standard" function syntax is more verbose - but I prefer code that is verbose but explicit to code that shorter but less explicit: if it's a function I like to explicitly call it `function` 😛 

But it's just my taste, we did not define a standard for the project